dmutils/errors.py

```diff
@@ -50,6 +50,10 @@ def render_error_page(e=None, status_code=None):
         503: "errors/500.html",
     }
 
+    # Handle exceptions with .status_code, not .code (e.g. dmapiclient.HTTPError)
+    if hasattr(e, 'status_code'):
```

Comment:
I think this line might raise an exception if you supply render_error_template with a status_code instead of an exception; I couldn't see any unit tests where that path is tested though

Reply:
Erk, good spot. Can you tell I did this in a rush 😹

Reply:
After all that this line would have worked fine with `None` 😂

Reply:
I like the new way better! 😎